react/features/analytics/handlers/AmplitudeHandler.js

```diff
@@ -0,0 +1,71 @@
+import amplitude from 'amplitude-js';
+
+import { getJitsiMeetGlobalNS } from '../../base/util';
+
+import AbstractHandler from './AbstractHandler';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Analytics handler for Amplitude.
+ */
+class AmplitudeHandler extends AbstractHandler {
+    /**
+     * Creates new instance of the Amplitude analytics handler.
+     *
+     * @param {Object} options -
+     * @param {string} options.amplitudeAPPKey - The Amplitude app key required
+     * by the Amplitude API.
+     */
+    constructor(options) {
+        super();
+
+        const { amplitudeAPPKey } = options;
+
+        if (!amplitudeAPPKey) {
+            logger.warn(
+                'Failed to initialize Amplitude handler, no tracking ID');
+
+            return;
+        }
+
+        this._enabled = true;
+
+        amplitude.getInstance().init(amplitudeAPPKey);
+    }
+
+    /**
+     * Sets the Amplitude user properties.
+     *
+     * @param {Object} props - The user portperties.
+     * @returns {void}
+     */
+    setUserProperties(props) {
```

Comment:
It does nothing in case of disable state. Is it obvious ? `return wasChaned = this._isEnabled`

Reply:
Since we don't currently need to use any result from the function I prefer to leave the code as it is. Maybe I can put some text about it in the JSDocs?